src/main/kotlin/com/sourcegraph/cody/ui/AttributionButtonController.kt

```diff
@@ -0,0 +1,55 @@
+package com.sourcegraph.cody.ui
+
+import com.intellij.openapi.project.Project
+import com.intellij.util.concurrency.annotations.RequiresEdt
+import com.sourcegraph.cody.agent.CurrentConfigFeatures
+import com.sourcegraph.cody.agent.protocol.AttributionSearchResponse
+import com.sourcegraph.cody.attribution.AttributionListener
+
+class AttributionButtonController(val button: ConditionalVisibilityButton) : AttributionListener {
+
+  private val extraUpdates: MutableList<Runnable> = ArrayList()
+
+  companion object {
+    fun setup(project: Project): AttributionButtonController {
+      val button = ConditionalVisibilityButton("Attribution search")
```

Comment:
What about the chat persistence? when I reopen the chat after IDE restart I see this text only 😕

Reply:
Filed https://github.com/sourcegraph/sourcegraph/issues/60229